learn/reference/variables/built-in.mdx

```diff
@@ -13,7 +13,7 @@ Here's a list of all the built-in variables in Botpress:
 | `conversation.SummaryAgent.transcript`                                          | Agent       | Transcript of the conversation going back a certain amount of turns.                                                                                                                                                                         |
 | `turn.KnowledgeAgent.answer`                                                    | Agent       | The answer provided by the [Knowledge Agent](/learn/reference/agents/knowledge-agent)                                                                                                                                                                 |
 | `turn.KnowledgeAgent.responded`                                                 | Agent       | Whether the Knowledge Agent has responded automatically to a user question. Its value will be `true` or `false`                                                                                                                              |
-| `turn.KnowledgeAgent?.answer?.length`                                           | Agent       | The amount of characters in the Knowledge answer. Use this in an [Expression](/learn/reference/cards/flow-logic#expression) to check if there is an answer in the KB. Optionally add `!` to start of the code in order to check if there was NOT an answer. |
+| `turn.KnowledgeAgent?.answer?.length`                                           | Agent       | The amount of characters in the Knowledge answer. Use this in an [Expression](/learn/reference/cards/flow-logic#expression) to check if there is an answer in the KB. Optionally add `!` to start of the code in order to check if there WASN'T an answer. |
```

Comment:
vale's suggestions for these things isn't great. It needs to understand case changes...